### PR TITLE
[ELY-2647] Add link to elytron-examples in README

### PR DESCRIPTION
**Pull request by Akshayabhat**
Ticket: https://issues.redhat.com/browse/ELY-2647

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,11 @@ Our [contribution guide](https://github.com/wildfly-security/wildfly-elytron/blo
  
 For more details, check out our [getting started guide](https://wildfly-security.github.io/wildfly-elytron/getting-started-for-developers/) for developers.
 
+Example Feature demos
```
Comment (SoniaZaldana) [inline README.md]: Thanks @Akshayabhat, just a small nitpick :) Could we substitute ```demos``` for ```Demos``` to stay consistent with the document capitalization.

File: README.md [hunk 2]
```diff
@@ -39,6 +39,11 @@ Our [contribution guide](https://github.com/wildfly-security/wildfly-elytron/blo
...
+---------------------
+
+Our [elytron-examples](https://github.com/wildfly-security-incubator/elytron-examples) repo contains example demos of WildFly Elytron features.
```
Comment (SoniaZaldana) [inline README.md]: Small nitpick here too! Could we substitute ```repo -> repository```?

**Review by SoniaZaldana — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Akshayabhat) [reply README.md]: yep, good catch! Updated!

File: README.md (hunk 2 above)
Comment (Akshayabhat) [reply README.md]: updated, thanks!

**Comment by SoniaZaldana** [comment]
Thank you @Akshayabhat! 

Last steps would be squashing both commits into one. You can reference the instructions I sent in the channel if you need any assistance :smile:

**Comment by skyllarr** [comment]
Hi @Akshayabhat , this looks great! It just need squashing of commits into a single commit as @SoniaZaldana mentioned. Please let us know if you need any help with that

**Comment by Akshayabhat** [comment]
> Thank you @Akshayabhat!
> 
> Last steps would be squashing both commits into one. You can reference the instructions I sent in the channel if you need any assistance 😄

Thanks, I just force pushed the changes under a single commit, hope that's ok!

**Comment by skyllarr** [comment]
@Akshayabhat Just a minor, please add an issue number to the beginning of the commit message. You can do that eg. by using `git commit --amend`  . Thanks!

**Comment by PrarthonaPaul** [comment]
Hi @Skyllarr 
This PR can be closed. Thanks!

**Pull request closed without merging**